Content/posts/team-restructure.md

```diff
@@ -0,0 +1,77 @@
+
+## Our Aims
+
+![Team Structure diagram. A discovery team with arrows for feature 1 pointing at feature team 1, feature 2 pointing at feature team 2, and feature 3 pointing at feature team 3. The discovery team contains a tech lead, product managers, performance analyst, user researcher and designers. Each of the three feature teams contains a tech lead, product manager, business analyst and Android, iOS, Typescript and Platform developers as well as automation and manual QA.](../../Images/team-restructure-after.png)
+*Our New Team Structure — Teams are split by business domain*
+
+When restructuring our teams, we wanted to solve as many of these irritations as possible.
+We created each of our teams around an end-to-end component of our app, with full ownership of their component.
+We have drawn the lines between the teams responsibilities with the aim of minimising cross-team dependencies.
+We wanted to ensure they have all skills to be able to successfully manage this _within_ the team, including making day-to-day decisions on maintenance and any enhancements.
+We’re hoping that this will enable the team to have a full understanding of their area and current work without being distracted by work going on in other parts of the app.
+Our leadership team should be an escalation path for the team to use at their discretion when problems arise.
+They are not there to micromanage the team or step in to make decisions on the team’s behalf.
+
+## Things we want to avoid
+
+Spreading members of each specialism across multiple teams could result in these people becoming isolated and not feeling sufficiently supported in their work.
+We’re aware that some of our less experienced developers are now the most senior developer of their particular specialism in their respective teams.
+We’ve attempted to mitigate this by putting them alongside a more experienced developer from another specialism.
+As part of this, we’re encouraging our senior developers to have a basic knowledge of the whole stack, not just their own code – after all general engineering principles should carry across these specialisms.
+
+> ”Individuals and interactions over processes and tools.“
+>
+> — Manifesto for Agile Software Development, [agilemanifesto.org](http://agilemanifesto.org)
+
+While we want to keep as much of our work within our team silos, we know that there will be occasions where our people will want advice and support from others in their specialism.
+We’re encouraging our teams to keep their regular specialism specific catch-up calls to help maintain the cross-team relationships that we're carrying over from our previous teams.
+We don’t want to go much further than this and over-define any inter-team ways of working preferring to follow the Agile Manifesto and trust individuals and interactions over processes.
+
+Our teams have raised concerns about loss of responsibility
+
+## Opportunities
```

Comment:
More freedom to decide _how_ to deliver a requirement or solve a problem. When the team has everything it needs to be able to do that, it can just get on with it independently.

Reply:
Yes– that's awesome, thanks! I've added it 😊